src/lib/annotations/doc/DocDrawingThread.js

```diff
@@ -116,16 +114,23 @@ class DocDrawingThread extends DrawingThread {
      * @return {void}
      */
     saveAnnotation(type, text) {
-        super.saveAnnotation(type, text);
+        this.emit('threadcleanup');
```

Comment:
Clean-up removes the listeners, 'threaddeleted' deletes the thread from the thread map.